staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/validation/formats.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"strings"
+
+	"github.com/go-openapi/spec"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+var supportedFormats = sets.NewString(
+	"bsonobjectid", // bson object ID
```

Comment:
why are there items in this map that are not documented?

Reply:
fixed